tests/distributions/test_transform.py

```diff
@@ -345,56 +345,13 @@ def check_transform_elementwise_logp(self, model):
             .sum()
             .eval({x_val_untransf: test_array_untransf})
         )
-        close_to(v1, v2, tol)
+        log_jac_det_eval = jacob_det.sum().eval({x_val_transf: test_array_transf})
```

Comment:
Testing the jacobian provides a regression check

Reply:
I think that you might be missing some edge cases if you `sum` the `jacob_det` and the other `logp`s. You might want to be pedantic with the test for the batch shapes and core shapes.